src/standalone/Bucket/Bucket.jl

```diff
@@ -526,7 +526,7 @@ function next_albedo(
 ) where {FT}
     (; α_snow, α_bareground) = model_albedo
     (; σS_c) = parameters
-    σS = Y.bucket.σS
+    σS = max.(Y.bucket.σS, FT(0))# if σS is small and negative, clip to zero for albedo estimation
```

Comment:
Should this be eps(FT), so it's consistent with the heaviside function? (also below)

Reply:
Ill change this to be consistent.

Reply:
I meant more for the case of `0 < σS < eps(FT)`, whether it should be treated as 0 or nonzero. It also appears in `partition_surface_fluxes()`. That said, it looks like the heaviside is only used for `snow_cover_fraction` which is then only used to scale evaporation, so I think that's ok, as long as the user doesn't get confused how to treat small values of snow. Maybe a comment would do. :)

Reply:
But in that case, should the comment on Line 457 (`# Equation (3) of the text, snow melt already multipied by snow_cover_fraction`) say `max(0, σS)` 🤔

Reply:
ah, I think I understand. You're right that we are being inconsistent in the `0 < σS < eps(FT)` regime. In principle, the snow_cover_fraction function can be anything (e.g. sigmoid function), and clipping negative values of σS to zero should still be a valid thing to do (they shouldnt depend on each other). I think they do because some of parameterizations depend on σS (e.g. the albedo, the beta function) and some depend on snow cover fraction. `partition_surface_fluxes` uses both in the same function.

Here is what I did to try and fix this:
- `partition_surface_fluxes` now works in units of emitting/absorbing area (snow). This is consistent with how we handle fluxes in the documentation and everywhere else - they are fluxes per emitting area and we multiply by emitting area fraction (1-σ) or σ to get fluxes per unit ground area when used in setting tendencies. σ is now applied outside the partition_surface_fluxes now, in Bucket.jl lines 457, 468, 474, and 437.
- The albedo model (linear interpolation between snow and bareground albedo) fundamentally conflicts with a heaviside function for snow cover fraction, so I left this alone for now. The inconsistency remains but it is O(eps) and doesnt affect any conservation. I left a note in the documentation.
- The other inconsistency arises in the beta function, but this function is independent of the snow cover fraction and implemented for numerical reasons only, so I dont know that they need to be consistent. if 0 < σS < eps(FT), this function wont be invoked for snow because we will use the beta of soil instead.

If we want to use the bucket model as a stepping stone for the full land model for testing different fractional coverages within a grid cell, we can make this consistent and use the SCF for the albedo function, at least.

Reply:
That seems reasonable to me. Thank you for digging into this, Kat!